cindy/index.js

```diff
@@ -0,0 +1,106 @@
+// 38-PB-program-switch-multiplechoice
+// Switch if...
+// 1. Color Analyzer
+
+// Create a switch statement that prints a different statement for each color possibility (red, blue, green, yellow).
+
+function colors(myColor){
+    switch(myColor){
+  
+      case 1: 
+      case 2: 
+      case 3: return "Sexy red";
+      case 4: 
+      case 5: return 'Peaceful blue';
+      case 6: 
+      case 7: 
+      case 8: return 'Wild green';
+      case 9: 
+      case 10: 
+      case 11: 
+      case 12: return 'Funny yellow';
+      default: return " only from 1 - 12 :c";
+    }  }
+  console.log("choose your color according to your month of birth  => ", colors(13))
```

Comment:
If I try to choose 1st, 2nd or 3rd case the result is always case3 because your declared 2 empty cases.
So it has no benefits to declare an empty case it just makes your program slower, it could be also better to use break after every case! Therefore I would prefer to get rid of empty cases!

Reply:
done!